CI: zstd compression via header doesn't work

example:

```
   9) All endpoints
       (/alignedAminoAcidSequences/S)
         should return zstd compressed data when accepting compression in header:
     Error: AssertionError: expected [ 62, 107, 101, 121 ] to deeply equal [ 40, 181, 47, 253 ]
ArrayBuffer content (for context): >key_1408408
MFVFLVLLPLVSSQCVNLTTRTQLPPAYTNSFTRGVYYPDKVFRSSVLHSTQDLFLPFFSNVTWFHAI--SGTNGTKRFDNPVLPFNDGVYFASTEKSNIIRGWIFGTTLDSKTQSLLIVNNATNVVIKVCEFQFCNDPFLGV-YHKNNKSWMESEFRVYSSANNCTFEYVSQPFLMDLEGKQGNFKNLREFVFKNIDGYFKIYSKHTPINLVRDLPQGFSALEPLVDLPIGINITRFQTLLALHRSYLTPGDSSSGWTAGAAAYYVGYLQPRTFLLKYNENGTITD...
      at expectIsZstdEncoded (test/common.ts:94:11)
      at Context.<anonymous> (test/common.spec.ts:222:9)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)

```

all the tests with "should return zstd compressed data when accepting compression in header" are failing in CI recently.

This commit on main has a green test: https://github.com/GenSpectrum/LAPIS/commit/f911c7106c787795a1b1bd7c01e95cb45b6aef54

I tried with LAPIS 0.6.1 and the same SILO version that was used back then (`commit-a1dcc46e52bd5e9d439be49b1acbb50a6898aeb0`) but I still get the failure: https://github.com/GenSpectrum/LAPIS/actions/runs/19143938994/job/54716439848?pr=1422

I don't get it locally at all, I tried a few different version combinations.

More CI tests I did in this PR: https://github.com/GenSpectrum/LAPIS/pull/1422 -- It also includes the little snippet which prints the clear text (see above error message)

---

I've made a PR to disable the test in CI for now, until we can sort this out: https://github.com/GenSpectrum/LAPIS/pull/1431

--- a/lapis-e2e/test/common.spec.ts
+++ b/lapis-e2e/test/common.spec.ts
@@ -219,7 +219,16 @@ describe('All endpoints', () => {
         expect(response.status).equals(200);
         expect(response.headers.get('content-type')).to.equal(expectedContentType(route.servesType));
         expect(response.headers.get('content-encoding')).equals('zstd');
-        expectIsZstdEncoded(await response.arrayBuffer());
+
+        // fetch automatically decompresses zstd responses as of node 24
+        if (route.servesType === 'SEQUENCES') {
+          expect(await response.text()).to.match(/^>key_/);
+        } else if (route.servesType === 'TREE') {
+          expect(await response.text()).to.match(/NODE_\d+;$/);
+        } else {
+          const body = await response.json();
+          expect(body.data).is.an('array');
+        }
       });
 
       it('should return gzip compressed data when asking for compression', async () => {

